src/polar/PolarAngleAxis.tsx

```diff
@@ -143,9 +144,12 @@ export class PolarAngleAxis extends PureComponent<Props> {
         y: lineCoord.y2,
       };
 
+      const className =
+        !React.isValidElement(tick) && !isFunction(tick) && typeof tick !== 'boolean' ? tick.className : '';
```

Comment:
Sorry, could you pull this out into a shared typeguard utility rather than repeating it?

Reply:
@ckifer 
How about this?
`PolarUtils.ts`
```ts
export const tickHasClassName = (
  tick?: SVGProps<SVGTextElement> | ReactElement<SVGElement> | ((props: any) => ReactElement<SVGElement>) | boolean,
) => !isValidElement(tick) && !isFunction(tick) && typeof tick !== 'boolean';
```
`PolarRadiusAxis.tsx`
```tsx
      return (
        <Layer
          className={clsx('recharts-polar-radius-axis-tick', tickHasClassName(tick) ? tick.className : '')}
          key={`tick-${entry.coordinate}`}
          {...adaptEventsOfChild(this.props, entry, i)}
        >
          {PolarRadiusAxis.renderTickItem(tick, tickProps, tickFormatter ? tickFormatter(entry.value, i) : entry.value)}
        </Layer>
      );
```